app/views/tags/index.html.erb

```diff
@@ -1,28 +1,28 @@
-<% title "Tags on #{community_name}" %>
+<% title t("core.tags_title", community_name: community_name) %>
```

Comment:
Let's just change this entirely to `<% title t("core.tag").pluralize %>`.

The logic is that if community name isn't automatically present, it defaults to `" — #{community_name}"`... So the current behavior here is too much of a special snowflake which can be changed to just "Tags", which will show up as "Tags — DEV Community".

Reply:
Thanks @benhalpern I'll make that change. Is there someone on the core team who speaks French or should I seek help on those translations?

Reply:
Let's please use the proper I18n mechanism for this: https://guides.rubyonrails.org/i18n.html#pluralization

Reply:
@citizen428 That mechanism doesn't seem to work (at least for the word "Tag")

`<% title t("core.tag", count: 2) %>`
<img width="231" alt="image" src="https://user-images.githubusercontent.com/1904364/135874835-68c29941-8192-42f6-9c07-bb5c9aa851aa.png">

Reply:
@lsparlin Because nobody added the necessary translations for it yet. If you do it works as expected

```yml
tag:
  one: "Tag"
  other: "Tags"
```

![image](https://user-images.githubusercontent.com/47985/136147804-b6e5a63d-ddee-49a5-bc95-a2415b611751.png)

But like I said, in this specific case I'd probably just go for a plural "core.tags" key and keep the other option around for  later to do things like "No tags", "1 tag", "x tags".